wsmaster/che-core-api-devfile/src/main/resources/schema/devfile.json

```diff
@@ -162,10 +162,7 @@
                 }
               },
               "then": {
-                "required": [
```

Comment:
Could you explain the reason why this `required` declaration can no longer be here? Also, why does the `id` have to be specified for both the editor and plugin when we have this common declarations block here?

Reply:
Good catch. `required` can still be here. Changed